feathr_project/feathr/__init__.py

```diff
@@ -45,6 +45,7 @@
     'MaterializationSettings',
     'MonitoringSettings',
     'RedisSink',
+    'HdfsSink',
```

Comment:
i am curious why materialize test can pass without this., does that means HdfsSink already included in other packages?